changelog/16.0/16.0.0/release_notes.md

```diff
@@ -101,11 +102,21 @@ or
 > ALTER TABLE your_table ENGINE=InnoDB;
 ```
 
-#### <a id="vttablet-restore-metrics">VTTablet Restore Metrics
+### <a id="vttablet-restore-metrics">VTTablet Restore Metrics
 
 As part of the VTTablet Sidecar Schema Maintenance Refactor in v16.0.0, we dropped the `local_metadata` table from the sidecar database schema. This table was storing a couple of metrics related to restores from backup, which have now been lost.
 They have been re-introduced in v17.0.0 as metrics that can be accessed from `/debug/vars`.
 
+### <a id="schema-init-upgrade"/>Schema-initialization stuck on semi-sync ACKs while upgrading to `v16.0.0`
+
+During upgrades from `v15.x.x` to `v16.0.0`, as part of `PromoteReplica` call, the schema-init realizes that there are schema diffs to apply and ends up writing to the database.
+The issue is that if semi-sync is enabled, all of these writes get blocked indefinitely.
+Eventually, `PromoteReplica` fails, and this fails the entire PRS call.
```

Comment:
Are these supposed to be part of the same paragraph, new paragraphs, or? Seems odd to me that there's a single newline between all three.

Reply:
I feel like we should also say `semi-sync` durability policy and link to the docs.